armi/settings/fwSettings/globalSettings.py

```diff
@@ -590,10 +590,10 @@ def defineSettings() -> List[setting.Setting]:
             description="Clean up files at the beginning of each cycle (BOC)",
         ),
         setting.Setting(
-            CONF_STATIONARY_BLOCKS,
-            default=[],
-            label="stationary Blocks",
-            description="blocks with these indices (int values) will not move in "
+            CONF_STATIONARY_BLOCK_FLAGS,
+            default=["GRID_PLATE"],
+            label="stationary Block Flags",
+            description="blocks with these flags (int values) will not move in "
```

Comment:
Add to the description that this is used in the fuel management so that it is not confused with load pads for vertical movements.